internal/cluster/validator.go

```diff
@@ -224,6 +224,36 @@ func (v *clusterValidator) isApiVipValid(c *clusterPreprocessContext) Validation
 	return boolValue(err == nil)
 }
 
+func (v *clusterValidator) isNetworkTypeValid(c *clusterPreprocessContext) ValidationStatus {
+	validNetworkTypes := []string{models.ClusterNetworkTypeOVNKubernetes, models.ClusterNetworkTypeOpenShiftSDN}
+	if !funk.ContainsString(validNetworkTypes, swag.StringValue(c.cluster.NetworkType)) {
+		return ValidationFailure
+	}
+	if network.IsIPv6CIDR(c.cluster.ClusterNetworkCidr) && swag.StringValue(c.cluster.NetworkType) != models.ClusterNetworkTypeOVNKubernetes {
+		return ValidationFailure
+	}
+	return ValidationSuccess
+}
+
+func (v *clusterValidator) printIsNetworkTypeValid(context *clusterPreprocessContext, status ValidationStatus) string {
+	switch status {
+	case ValidationSuccess:
+		return "The cluster has a sufficient network type."
+	case ValidationFailure:
+		validNetworkTypes := []string{models.ClusterNetworkTypeOVNKubernetes, models.ClusterNetworkTypeOpenShiftSDN}
+		if !funk.ContainsString(validNetworkTypes, swag.StringValue(context.cluster.NetworkType)) {
+			return "NetworkType is not valid, valid network types can be OpenShiftSDN or OVNKubernetes"
```

Comment:
```suggestion
			return "The network type is not valid; the valid network types are OpenShiftSDN or OVNKubernetes"
```

Also consider using the validNetworkTypes variable in the string